From e58b258ff7dd8cfac265030e009f4f382e322f4e Mon Sep 17 00:00:00 2001
From: Alexey Puzhevich <alexey.puzhevich@intel.com>
Date: Mon, 16 Jun 2025 15:48:52 +0300
Subject: [PATCH] [JSW-934] jgs: run in simulation environment

The needed changes were introduced to enable ppboot
execution in simulation environment (Coral) on mwm_ww19_4 spec.

Change-Id: I822e22c561c5fb8d308d5872c3fa3bcd44c23642
Signed-off-by: Alexey Puzhevich <alexey.puzhevich@intel.com>
---

diff --git a/board/habanalabs/jgs/post-image-spi.sh b/board/habanalabs/jgs/post-image-spi.sh
index a7fd635..ea42988 100755
--- a/board/habanalabs/jgs/post-image-spi.sh
+++ b/board/habanalabs/jgs/post-image-spi.sh
@@ -23,13 +23,13 @@
 PPBOOT_SEC_OFFSET=$((PPBOOT_PRI_OFFSET + SECONDARY_MANIFEST_OFFSET))
 PPBOOT_TEXT_BASE=0x300007FF8640000
 PPBOOT_START_ADDR=0x300007FF8640000
-PPBOOT_ALT_DMA_ADDR=0x300007FF8A08000
+PPBOOT_ALT_DMA_ADDR=0x300007FF9DF8000
 
 PREBOOT_PRI_OFFSET=0x30000
 PREBOOT_SEC_OFFSET=$((PREBOOT_PRI_OFFSET + SECONDARY_MANIFEST_OFFSET))
-PREBOOT_TEXT_BASE=0x300007FF8A08000
-PREBOOT_START_ADDR=0x300007FF8A08000
-PREBOOT_CPU_BASE_ADDR=0x79CE0000	# mmIO_0_CFG_HALF_0_PSOC_ARC_MNG_ARC_BASE
+PREBOOT_TEXT_BASE=0x300007FF9DF8000
+PREBOOT_START_ADDR=0x300007FF9DF8000
+PREBOOT_CPU_BASE_ADDR=0x786C0000	# mmIO_0_CFG_HALF_0_PSOC_ARC_MNG_ARC_BASE
 PREBOOT_IMAGE_FLAGS=0x7E		# load and exec for all rst types
 
 FAKE_FUSE_PRI_OFFSET=0x140000
